.github/workflows/auto-label.yml

```diff
@@ -0,0 +1,13 @@
+name: 🤖 Auto label
+on: pull_request_review
+
+jobs:
+  label-when-approved:
+    name: Label when approved
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3.6.0
+      - uses: taiga-family/ci/actions/variables@1.11.1
+      - uses: taiga-family/ci/actions/label-when-approved@1.11.1
```

Comment:
We need only one approve inside `Maskito` to merge PR.

Do we need this workflow ?

Reply:
fixed